docs/configuration/entrypoints.md

```diff
@@ -120,6 +120,14 @@ To redirect an entrypoint rewriting the URL.
 !!! note
     Please note that `regex` and `replacement` do not have to be set in the `redirect` structure if an entrypoint is defined for the redirection (they will not be used in this case).
 
+`regex` and `replacement` follow the rules as per [matchers](/basics/#matchers)
+
+Specifically the `replacement` rule uses [Go Regexp.ReplaceAllString](https://golang.org/pkg/regexp/#Regexp.ReplaceAllString)
+
+Care should be taken when defining replacement expand variables; `$1x` is equivalent to `${1x}`, not `${1}x` [see Regexp.Expand](https://golang.org/pkg/regexp/#Regexp.Expand)
```

Comment:
could you replace the `;` by `:`

Reply:
Done, plus added your Regex101 example from the original issue (missed that you had added that as I only saw the first github email notification)